google/cloud/security/common/data_access/group_dao.py

```diff
@@ -25,28 +25,68 @@
 class GroupDao(dao.Dao):
     """Data access object (DAO) for Groups."""
 
-    def get_group_users(self, resource_name, timestamp):
-        """Get the group members who are users.
+    def get_group_id(self, resource_name, email, timestamp):
```

Comment:
nit: s/email/group_email/

This is because it's stored that way in the database.